specification/purview/data-plane/readme.md

```diff
@@ -122,6 +122,21 @@ input-file:
 - Azure.Analytics.Purview.Share/preview/2021-09-01-preview/share.json
 ```
 
+``` yaml
+openapi-type: data-plane
+tag: package-2022-11-01-preview
+title: PurviewPDSClient
+```
+
+### Tag: package-2022-11-01-preview
+
+These settings apply only when `--tag=package-2022-11-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2022-11-01-preview'
+input-file:
+- Azure.Analytics.Purview.PDS/preview/2022-11-01-preview/pds.json
+```
+
```

Comment:
If the various components of Purview are going to version and release independently we should probably split this readme into separate files, one for each component in each component's subdirectory.

Reply:
Sure, we will add a new readme file for the GA Version.
For this preview version can we keep in a common file?
Resolving this conversation for now. Let me know if this is fine.